docs/source/architecture/servicefees.rst

```diff
@@ -0,0 +1,83 @@
+.. _service-fees:
+
+Service Fees (Pricing)
+======================
+
+Minimum fee rate
+----------------
+
+When they join the network, stakers specify a minimum fee rate, on a *per sharing policy* and *per 24h period* basis, that their worker (Ursula) will accept at the point of engagement with a network user (Alice). If Alice’s offered per period rate (contained alongside the sharing policy’s parameters in an ``Arrangement`` object) for a specified policy duration computes as equal to or greater than the minimum fee rate, the sharing policy will be accepted by Ursula and the access control service will commence. Ursula will service the sharing policy by being online and answering access requests, at that unchanging per period fee rate, until the specified expiration date or an early revocation is instigated by Alice.
+
+The minimum fee rate applies to each individual worker machine managing a given sharing policy. In other words, the rate is also *per Ursula*. 
+
+If Alice wishes to employ multiple Ursulas to service a single sharing policy (``n`` > 1), a common configuration, then they must pay each staker associated with that policy the same exact fee rate. This rate will be the highest from the set of *minimum* fee rates specified by the stakers with which they engage. Alices may attempt price optimization strategies to find the cheapest group of Ursulas.
+
+When issuing a sharing policy, Alices are required to escrow a deposit covering the cost of the entire duration of the policy. The deposit is paid out in tranches to stakers once per period, in the period after their Ursula makes a commitment (formerly referred to as *confirm activity*). Therefore, the minimum sum of fees a staker can receive each period is the product of their specified minimum fee rate and the number of active sharing policies their Ursula has been assigned.
```

Comment:
*in the period after their Ursula makes a commitment* -> slightly wrong
1st period - Ursula makes a commitment
2nd period -> this period
3rd period -> staker can get fee for policy for 2nd period

Reply:
```suggestion
When issuing a sharing policy, Alices are required to escrow a deposit covering the cost of the entire duration of the policy. This deposit is split and paid out in tranches to stakers once per period. 

The precise payment flow follows a repeated three-period cycle; in the first period the Ursula *makes a commitment* to the second period. Then, the Ursula services the policy during the second period (and makes a commitment to the third period). In the third period, they receive the fee tranche for their work during the second period, and continue servicing the policy, etc. This cycle continues until all the policies that Ursula is responsible for have expired.

The minimum sum of fees a staker can receive period-to-period is the product of their specified minimum fee rate and the number of active sharing policies their Ursula has been assigned. 

Note that *making a commitment* was formerly referred to as *confirming activity*.
```